python/dgl/graphbolt/impl/ondisk_dataset.py

```diff
@@ -129,14 +129,41 @@ def preprocess_ondisk_dataset(
                     graph_feature["format"],
                     in_memory=in_memory,
                 )
-                g.ndata[graph_feature["name"]] = node_data
+                if is_homogeneous:
+                    g.ndata[graph_feature["name"]] = node_data
+                else:
+                    g.nodes[graph_feature["name"]] = {
```

Comment:
is this workable? the canonical way is `g.nodes[NODE_TYPE].data['x'] = xxx`?

Reply:
I see, I will change to the canonical solution. But my solution is workable as in https://docs.dgl.ai/en/latest/generated/dgl.DGLGraph.ndata.html.

Reply:
Oh, should be `g.ndata` instead of `g.nodes`. This is not tested in TGIF.

Reply:
I'm ok with `g.ndata`.